x-pack/plugins/ml/public/jobs/jobs_list_new/jobs.html

```diff
@@ -0,0 +1,11 @@
+<ml-nav-menu name="jobs"></ml-nav-menu>
+<ml-jobs-page-wrapper>
+  <div class='jobs-stats-bar'>
+    <span ng-if='js.show' class='stat' ng-repeat='js in jobStats'>
+      <span class='stat-label'>{{js.label}}</span>: <span class='stat-value'>{{js.value}}</span>
+    </span>
+  </div>
+  <!-- <ml-message-bar ></ml-message-bar> -->
```

Comment:
Is the message bar no longer needed, or is the plan to replace it with an EUI component?